Docs/Whats_New.md

```diff
@@ -16,22 +16,30 @@ For best results, use a Markdown reader.*
 MoltenVK 1.0.33
 ---------------
 
-Released TBD
+Released 2019/02/28
 
 - Support the `VK_EXT_memory_budget` extension.
+- Support  8-bit part of `VK_KHR_shader_float16_int8`.
+- Disable the `shaderStageImageMultisample` feature.
```

Comment:
This should be "Disable the `shaderStorageImageMultisample` feature. That was a typo on my part.

Reply:
And clearly...lazy copy-and-paste editing on my part! 😉

Thanks for noticing.